cmd/vineyard/app/vineyard.go

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Could you help keep the same format with the followings:

```golang
/*
Copyright 2023 The Fluid Authors.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

Reply:
Thanks for the advice, I have updated the lincense from 2023 to 2024.